internal/config.go

```diff
@@ -36,6 +37,22 @@ const (
 	DefaultClusterName    = "dev"
 )
 
+type SSLConfig struct {
+	ServerName         string
+	InsecureSkipVerify bool
+	CAPath             string
+	CertPath           string
+	KeyPath            string
+	KeyPassword        string
+}
+
+type Config struct {
+	Hazelcast hazelcast.Config
+	SSL       SSLConfig
```

Comment:
Why these are next to each other and SSL is not child of Hazelcast?

Reply:
Go client doesn't support most of the SSL settings declaratively, so I had to add an extra config section for CLC.